CREDITS.md

```diff
@@ -34,18 +34,6 @@ Graphics
 
 ### Character Sprites
 
-* ["Player Sprite"](https://git.tuxemon.org:3000/tuxemon/tuxemon/blob/development/resources/sprites/player_front.png)

```

Comment:
Is there a reason why you are removing this attribution from the credits? Mike Bramson was the artist who created the player sprite that you reverted back to. In general, I would avoid removing people from the credits. If it's unused, we can put their attribution in another section of the credits to attribute people who helped make intermediate artwork.